followthemoney/property.py

```diff
@@ -1,36 +1,36 @@
-from normality import stringify
+from normality import stringify  # type: ignore
 from banal import is_mapping
-from rdflib import URIRef
+from rdflib import URIRef  # type: ignore
+from typing import Dict, Any, Mapping, Optional
 
+from followthemoney.model import Model
+from followthemoney.schema import Schema
 from followthemoney.exc import InvalidModel
+from followthemoney.types.common import PropertyType
 from followthemoney.types import registry
 from followthemoney.util import gettext, NS, get_entity_id
 
 
 class Property(object):
     RESERVED = ['id', 'caption', 'schema', 'schemata']
 
-    def __init__(self, schema, name, data):
-        self.schema = schema
-        self.model = schema.model
+    def __init__(self, schema: Schema, name: str, data: Mapping[str, Any]):
+        self.schema: Schema = schema
+        self.model: Model = schema.model
 
-        self.name = stringify(name)
-        self.qname = '%s:%s' % (schema.name, self.name)
+        self.name: str = stringify(name)
+        self.qname: str = '%s:%s' % (schema.name, self.name)
         if self.name in self.RESERVED:
             raise InvalidModel("Reserved name: %s" % self.name)
 
         self.data = data
-        self._label = data.get('label', name)
-        self._description = data.get('description')
-        self.hidden = data.get('hidden', False)
-        self.stub = data.get('stub', False)
+        self._label: str = data.get('label', name)
+        self._description: Optional[str] = data.get('description')
+        self.hidden: bool = data.get('hidden', False)
+        self.stub: bool = data.get('stub', False)
 
-        type_ = data.get('type', 'string')
-        self.type = registry.get(type_)
-        if self.type is None:
-            raise InvalidModel("Invalid type: %s" % type_)
```

Comment:
See registry, weird behaviour change.

Reply:
Yeah, I did that because get now always returns a `PropertyType`, or fails. See comment over there.